kitsune/search/tasks.py

```diff
@@ -121,7 +121,7 @@ def index_chunk_task(write_index, batch_id, chunk):
 
     except Exception:
         rec.text = u'{0}: Errored out {1} {2}'.format(
-            rec.text, sys.exc_type, sys.exc_value)
+            rec.text, sys.exc_type, sys.exc_value)[:255]  # Truncate at 255 chars.
```

Comment:
`log.exception()` might be a good idea. I don't know where it goes, though.

Also, I think I'd opt for increasing the size of the column instead.